# Prokaryotic-Evolutionary-Dynamics-in-a-SoCal-Coastal-Marine-Ecosystem (October-2024 to December-2024)

## DATA GENERATION:

- **Perform the following steps on your Mac Terminal (bash):**
    - [Phage Receptor Identification on OceanMAGs Database](https://www.notion.so/Phage-Receptor-Identification-on-OceanMAGs-database-1b4c2e3c8016806e882ce028c52d15b3?pvs=21)

- **Perform the following steps on CARC on-demand HPC:**
    - [Extract Matched Phage Receptor Accession Names and Create Custom Prokaryotic Protein Database](https://www.notion.so/Extract-Matched-Phage-Receptor-Accession-Names-and-Create-Custom-Prokaryotic-Protein-Database-1b4c2e3c801680dbbe22c6805a650aaf?pvs=21)
    - [Generate Custom Prokaryotic Nucleotide Database](https://www.notion.so/Generate-Custom-Prokaryotic-Nucleotide-Database-1b4c2e3c80168037b679da85e0d5c0e4?pvs=21)
    - [Downloading SRA Files Using NCBI SRA Toolkit](https://www.notion.so/Downloading-SRA-Files-Using-NCBI-SRA-Toolkit-1b4c2e3c8016806984f7f9251ef1d178?pvs=21)
    - [Map SRA Metagenomic Reads to Custom Prokaryotic Nucleotide Database Using BWA](https://www.notion.so/Map-SRA-Metagenomic-Reads-to-Custom-Prokaryotic-Nucleotide-Database-Using-BWA-1b4c2e3c80168070aa38d377f331b3f6?pvs=21)

## DATA ANALYSIS:

- **Perform the following steps on Google Colab Pro:**
    - [Normalization of Metagenomic Abundance Matrix (RPKM Calculation)](https://www.notion.so/Normalization-of-Metagenomic-Abundance-Matrix-RPKM-Calculation-1b4c2e3c801680779851ec2830a152c1?pvs=21)
    - [Filtering Metagenomic Abundance Matrix with 10X Depth Coverage Matrix](https://www.notion.so/Filtering-Metagenomic-Abundance-Matrix-with-10X-Depth-Coverage-Matrix-1b4c2e3c8016806f9d71c39eed4c5f07?pvs=21)

## DATA VISUALIZATION:

- **Perform the following steps on Google Colab Pro:**
    - [Visualizing Metagenomic Abundance Matrix Using a Heatmap Plot](https://www.notion.so/Visualizing-Metagenomic-Abundance-Matrix-Using-a-Heat-map-Plot-1b4c2e3c8016805cb04bd52d8a65ca53?pvs=21)
